packages/gatsby/cache-dir/loader.js

```diff
@@ -245,7 +305,16 @@ export class BaseLoader {
   loadPageSync(rawPath) {
     const pagePath = findPath(rawPath)
     if (this.pageDb.has(pagePath)) {
-      return this.pageDb.get(pagePath).payload
+      const pageData = this.pageDb.get(pagePath).payload
+      const staticQueryData = {}
+
+      pageData.staticQueryHashes.forEach(hash => {
+        staticQueryData[hash] = this.staticQueryDb.get(hash)
+      })
+      return {
+        ...pageData,
+        staticQueryData,
+      }
```

Comment:
shouldn't `staticQueryData` be part of entries stored in `this.pageDb`? Feels unnecessary to construct this every time on read when we can do it once on write?

Reply:
Yes, it is now stored in `pageDb`